contracts/Tokens/VTokens/VToken.sol

```diff
@@ -511,6 +533,55 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return _setInterestRateModelFresh(newInterestRateModel);
     }
 
+    /**
+     * @notice Accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
+     * @dev Admin function to accrue interest and update the stable interest rate model
+     * @param newStableInterestRateModel The new interest rate model to use
+     * @return uint 0=success, otherwise a failure (see ErrorReporter.sol for details)
+     * custom:events Emits NewMarketInterestRateModel event; may emit AccrueInterest
+     * custom:events Emits NewMarketStableInterestRateModel, after setting the new stable rate model
+     */
+    function setStableInterestRateModel(StableRateModel newStableInterestRateModel) public returns (uint256) {
+        // Check caller is admin
+        if (msg.sender != admin) {
+            return fail(Error.UNAUTHORIZED, FailureInfo.SET_STABLE_INTEREST_RATE_MODEL_OWNER_CHECK);
+        }
+
+        uint error = accrueInterest();
+        if (error != uint(Error.NO_ERROR)) {
```

Comment:
A reminder for doing this check in the IL version, if they make sense, here: https://github.com/VenusProtocol/isolated-pools/blob/feature/stable-borrowing/contracts/VToken.sol#L1844

Reply:
I will look into it